README.md

```diff
@@ -9,24 +9,24 @@ a Zip file containing a Maven project populated for an Eclipse Vert.x, Spring Bo
 container.
 
 * Build this project:
-
 ```bash
 $ mvn clean install
 ```
 
 * Set the following environment variables: 
-
 		$ export LAUNCHER_MISSIONCONTROL_SERVICE_HOST=localhost
 		$ export LAUNCHER_MISSIONCONTROL_SERVICE_PORT=8080
 
 * Execute the uber-jar in the target folder with:
-
 ```bash
-$ cd web
-$ mvn wildfly-swarm:run
+$ java -jar web/target/launcher-backend-swarm.jar
 ```
 
-Then follow the [front-end README](https://github.com/fabric8-launch/launcher-frontend/blob/master/README.md) to run the front-end.
+* Set the environment variable for the frontend to use your local version of the backend
+    
+    $ export LAUNCHER_BACKEND_URL=http://localhost:8080/api/
+
```

Comment:
I don't think this information should be here, but in the frontend README.

Reply:
And it should then also mention the required:

    $ export LAUNCHER_MISSIONCONTROL_URL=ws://localhost:8080

(in the same frontend README)